frontend/src/scenes/trends/LineGraph.js

```diff
@@ -112,7 +112,10 @@ export class LineGraph extends Component {
                                       )
                                           return null
                                       var label = data.datasets[tooltipItem.datasetIndex].label || ''
-                                      return label + ' - ' + tooltipItem.yLabel.toLocaleString()
+                                      let formattedLabel = label + ' - ' + tooltipItem.yLabel.toLocaleString()
+                                      if (formattedLabel.includes('Average Duration of Session'))
```

Comment:
This feels super brittle. Maybe this should just be a prop that you pass a function to (`labelFormat` or something), then we can pass that from ActionsLineGraph where at least we have access to the filter object

Reply:
I changed it so that the backend passes an extra `chartLabel` attribute in case we want to different labels for charts and tables. Let me know fi this works